src/api/app/models/project.rb

```diff
@@ -1750,6 +1753,13 @@ def has_local_distribution(project_name, repository)
         linked_repository.name == repository
     end
   end
+
+  def update_staging_workflow_on_backend
+    return if staging_workflow_id.nil?
```

Comment:
:see_no_evil: `return unless staging_workflow_id`

Reply:
It's not exactly the same, but I guess in this case, it will be fine.